samples/samples/snippets.py

```diff
@@ -1070,17 +1068,17 @@ def update_data_with_json(instance_id, database_id):
             values=[
                 (
                     4,
-                    json.dumps(
+                    JsonObject(
                         [
-                            {"name": "room 1", "open": True},
-                            {"name": "room 2", "open": False},
+                            JsonObject({"name": "room 1", "open": True}),
+                            JsonObject({"name": "room 2", "open": False}),
                         ]
                     ),
                 ),
-                (19, json.dumps({"rating": 9, "open": True})),
+                (19, JsonObject({"rating": 9, "open": True})),
```

Comment:
I think one of the `JsonObject`s here can be written in another style, like:
```python
JsonObject(rating=9, open=True)
```
With a comment that this style of arguments is also supported - some users may consider it more convenient.